### PR TITLE
Mobile branch

### PR DESCRIPTION
**Pull request by NelsonAraujo93**
In this pull request I followed the next steps:

**First Mobile**

- Create an index.html file.
- Create a styles.css file.
- Create a mobile navigator for mobile.
- Create a header section for mobile.
- Create a Program section for mobile.
- Create an Artists section for mobile.
- Create a Partners section for mobile.
- Create a footer.
- Create an about.html file.
- Create an about.js file.
- Create an About header.
- Create an Last festival section for about.hmtl file.
- Create a Logo section for about.html file.
- Add Alternative text to all images for enhance accessibility.

**Desktop**

- Create css rules to Mid and Big screens (Responsiveness).

**JavaScript**

- Create open and close menu functions in index.js and about.js.
- Create a function to load Artist Section Dynamically on index.html.
- Create a Scroll Listener function to the navigators on index.js and about.js.

**Linters**

- Check Both Css and JS linters.

**Deploy**

- Deploy the project in gitHub Pages.

**Git**

- Update README file.

**Loom Video**

- [Click here](https://www.loom.com/share/e06aac632cfa4e649294d79bd54917d6)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,7 +227,7 @@ I would like to thank Microverse for giving me the chance to improve my coding.
 
 - **What is this for?**
 
-  - This is a Microverse portfolio activity to help us learn how to create a nice portfolio and responsive website.
+  - This is a Microverse Module 1 Capstone activity to help us learn how to create a nice website with best practices.
```
Comment (tufoinnkuo10) [inline README.md:230]: - [x] nice work. please ensure that in your "Acknowledgements" section, you give  appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <title>Module 1 Capstone</title>
+</head>
+<body>
+  <nav id="main-navigator">
+    <div class="wrapper">
+      <ul id="nav-menu">
+        <li class="main-nav-item">
```
Comment (tufoinnkuo10) [inline index.html:39]: - [x] great job. you are missing the top dark nav bar for the home/about desktop version. The top short bar section with social media links, logout, and language links aligned to the right.
yours
Screenshot (706)
behance
Screenshot (707)_LI

File: index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+						<img src="./assets/spotify-logo.png" alt="Spotify Icon">
+					</li>
+          <li class="partner-item">
+						<img src="./assets/sony-logo.png" alt="Sony Music Icon">
+					</li>
+        </ul>
+      </div>
+    </section>
+    <footer class="white-background full-width">
+      <div class="logo-container">
```
Comment (tufoinnkuo10) [inline index.html]: - [x] please the footer contains a logo aligned to the left and text aligned to the right. yours kind of look centralized. please fix that a little. same for the about page as well


Screenshot (709)_LI

File: index.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+            <div class="logistic-item">
+              <div class="logistic-item-icon">
+                <img src="./assets/vynil.png" alt="Vynil Icon">
+              </div>
+              <div class="logistic-item-label lato-font-bold secondary-color">Clousure</div>
+              <div class="logistic-item-content lato-font third-color">Don't forget your bonus ticket for the 2024 festival.</div>
+            </div>
+          </li>
+        </ul>
+        <button class="main-btn secondary-background white-color lato-font-bold"> Join Us</button>
```
Comment (tufoinnkuo10) [inline index.html:130]: - [x] please stick as much as possible to the structure while personalizing it. The join us button appears only in the mobile version. Kindly add the link text like in the original design. please fix that
yours
Screenshot (710)_LI
behance
Screenshot (711)_LI

File: index.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
+const artistArray = [
+  {
+    name: 'Duke Elington',
```
Comment (tufoinnkuo10) [inline index.js:3]: - [x] Remember, you are supposed to follow the provided design guidelines for this project, but **personalize the content**.  Please ensure that in the desktop version, the first six articles/speakers/musicians are showing like in the original design, and you can add more for the extra see more function. 
yours
Screenshot (712)
behance
Screenshot (711)

File: about-us.html [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+      </div>
+    </section>
+    <footer class="fourth-background full-width">
+      <div class="wrapper">
+        <div class="logo-container">
+            <img src="./assets/logo-navbar.png" alt="Vagos Icon">
+            <h4 class="lato-font third-color">Vagos Studios</h4>
+        </div>
+        <div class="rigth-container">
+          <h5 class="lato-font third-color">
```
Comment (tufoinnkuo10) [inline about-us.html:134]: - [x] please the footer of the about page also has a white background with the logo aligned to the left and text aligned to the right. Kindly add that so it follows the original design
yours
Screenshot (714)
Behance
Screenshot (713)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @NelsonAraujo93 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Design well followed
✅ Readme well personalized

Required Changes ♻️
Check the comments under the review

video  presentation
- please always ensure to include the reason why you created a particular project in your presentation and how that project will be helpful to you/anyone around you or to the public at large. nice work

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @NelsonAraujo93, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**